x-pack/plugins/index_management/public/store/reducers/table_state.js

```diff
@@ -39,6 +40,15 @@ export const tableState = handleActions({
       showSystemIndices,
     };
   },
+  [toggleChanged](state, action) {
+    const { toggleName, toggleValue } = action.payload;
+    const toggles = { ...state.toggles };
+    toggles[toggleName] = toggleValue;
```

Comment:
Could we rename `toggles` to `toggleNameToVisibleMap` to make it clearer what this data structure represents?

Reply:
done